examples/Server/TicketRepository.cs

```diff
@@ -35,7 +35,7 @@ public int GetAvailableTickets()
             return _availableTickets;
         }
 
-        public bool BuyTickets(string user, int count)
+        public bool BuyTickets(string? user, int count)
```

Comment:
Looks like we resolved the nullable issue differently

Reply:
There will always be a user with a name because the method forces authorization. My fix is better 😋